    public void run(TestResult testResult) {
      testResult.addError(this, cause);
    }

